### PR TITLE
Added prototype

### PR DESCRIPTION
**Pull request by webbyfox**

### CODE REVIEW

File: djangocms_navigation/admin.py [hunk 1]
```diff
@@ -19,9 +22,84 @@ def save_model(self, request, obj, form, change):
             )
         super().save_model(request, obj, form, change)
 
+    def get_list_display(self, request):
```
Comment (chaosk) [inline djangocms_navigation/admin.py]: If you're not adding any logic to modify the contents of list_display, consider just setting list_display attribute

File: djangocms_navigation/admin.py [hunk 2]
```diff
@@ -19,9 +22,84 @@ def save_model(self, request, obj, form, change):
...
+        list_display = ["title", "get_menuitem_link"]
+        return list_display
+
+    def get_menuitem_link(self, obj):
+        object_preview_url = reverse(
+            "admin:{app}_menuitem_list".format(
```
Comment (chaosk) [inline djangocms_navigation/admin.py]:
```suggestion
             "admin:{app}_{model}_list".format(
```

File: djangocms_navigation/admin.py [hunk 3]
```diff
@@ -19,9 +22,84 @@ def save_model(self, request, obj, form, change):
...
+                app=obj._meta.app_label, model=obj._meta.model_name
+            ),
+            args=[obj.pk],
+        )
+
+        return format_html(
+            '<a href="{}" class="js-moderation-close-sideframe" target="_top">'
+            '<span class="cms-icon cms-icon-eye"></span> Items'
```
Comment (chaosk) [inline djangocms_navigation/admin.py]: Make "Items" translatable

File: djangocms_navigation/admin.py [hunk 4]
```diff
@@ -19,9 +22,84 @@ def save_model(self, request, obj, form, change):
...
+
+    def get_queryset(self, request):
+        if hasattr(self, "menu_content_id"):
+            return MenuItem.objects.filter(menu_content=self.menu_content_id)
+        return self.model().get_tree()
+
+    def add_view(self, request, menu_content_id=None, form_url="", extra_context=None):
+        extra_context = extra_context or {}
+        if menu_content_id:
+            self.menu_content_id = menu_content_id
```
Comment (chaosk) [inline djangocms_navigation/admin.py]: Can you set this on request instead?

File: djangocms_navigation/admin.py (hunk 2 above)
Comment (webbyfox) [reply djangocms_navigation/admin.py]: the model here is 'menucontent` hence I hardcorded to `menuitem`

**Pull request merged**